frontend/src/views/LoginView.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div>
+    <h1>LOGIN</h1>
+    <form
+      @submit.prevent="login"
+      style="border: solid 1px black; width: 25vw; min-width: 300px"
+      class="mx-auto p-3"
+    >
+      <div class="bg-danger text-white p-1 mx-4 rounded" v-if="erreurs.length">
+        <span v-for="erreur in erreurs" :key="erreur">{{ erreur }}</span>
+      </div>
+      <input
+        class="text-center p-1 my-3 rounded"
+        style="background-color: #e3e3e3"
+        v-model="email"
+        placeholder="Courriel"
+      />
+      <div style="width: 100%"></div>
+      <input
+        class="text-center p-1 rounded"
+        style="background-color: #e3e3e3"
+        v-model="motdepasse"
+        placeholder="Mot de passe"
+        type="password"
+      />
+      <div style="width: 100%"></div>
+      <button class="btn btn-success px-5 my-3" type="submit">Login</button>
+      <div style="width: 100%"></div>
+      <router-link
+        to="/signup"
+        class="text-secondary"
+        style="text-decoration: none"
+        >S'inscrire</router-link
+      >
+    </form>
+  </div>
+</template>
+<script>
+export default {
+  data: () => {
+    return {
+      email: "",
+      motdepasse: "",
+      erreurs: [],
+    };
+  },
+  methods: {
+    login() {
+      fetch("http://localhost:3000/auth/login", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          email: this.email,
+          motdepasse: this.motdepasse,
+        }),
+      })
+        .then((response) => {
+          if (response.status === 200) {
+            return response.json();
+          } else {
+            this.erreurs.push("Erreur !");
+          }
+        })
+        .then((data) => {
+          console.log("data", data);
```

Comment:
You should remove console statement as mentioned in the Github Action Warning juste below